configs/projects/cfacter.rb

```diff
@@ -29,17 +29,22 @@
   proj.setting(:project_version, gem_version)
   proj.setting(:gemdir, '/var/tmp/facter_gem')
   if platform.is_windows?
-    proj.setting(:ruby_dir, '/cygdrive/c/Program\\ Files/Puppet\\ Labs/Puppet/sys/ruby')
-    proj.setting(:gem_binary, 'cmd /c "C:\Program Files\Puppet Labs\Puppet\sys\ruby\bin\gem.bat"')
-    proj.setting(:ruby_binary, 'cmd /c "C:\Program Files\Puppet Labs\Puppet\sys\ruby\bin\ruby.exe"')
+    proj.setting(:artifactory_url, "https://artifactory.delivery.puppetlabs.net/artifactory")
+    proj.setting(:buildsources_url, "#{proj.artifactory_url}/generic/buildsources")
+    proj.setting(:ruby_dir, '/cygdrive/c/ProgramFiles64Folder/PuppetLabs/Puppet/sys/ruby')
```

Comment:
Will there be 32-bit builds for these gems? I'm thinking about `ProgramFilesFolder` vs `ProgramFiles64Folder` and whether that could make for any trouble - I don't really understand the windows process well enough to know, though.

Reply:
there may be in the future but for now lets stick with the 64 bit build.